mantidimaging/gui/windows/spectrum_viewer/view.py

```diff
@@ -371,3 +445,44 @@ def set_binning_visibility(self) -> None:
         self.bin_size_spinBox.setHidden(hide_binning)
         self.bin_step_label.setHidden(hide_binning)
         self.bin_step_spinBox.setHidden(hide_binning)
+
+    def set_roi_properties(self) -> None:
+        if not any([spinbox.setEnabled(False) for spinbox in self.roiPropertiesSpinBoxes.values()]):
```

Comment:
This causes ruff checks to fail under code: `C419 Unnecessary list comprehension`, I'd suggest changing to this to pass Ruff static analysis
```suggestion
    for spinbox in self.roiPropertiesSpinBoxes.values():
        spinbox.setEnabled(True)
```